datanode/entities/pagination.go

```diff
@@ -25,7 +25,11 @@ import (
 )
 
 const (
-	DefaultPageSize int32 = 1000
+	// DefaultPageSize is the default number of records that will be returned if no paging constraints
+	// are provided in an API request that supports pagination.
+	// If we set this too large, we get errors relating to gRPC message size so for now set it to
+	// something small and if users want more, they should set the appropriate limit in their request.
+	DefaultPageSize int32 = 50
```

Comment:
Can we confirm with frontend this isn't gonna break anything before merging?

Reply:
I've dropped a message in Slack to get some input from the FE team https://vegaprotocol.slack.com/archives/C8WS7M50E/p1708007031793479

Reply:
After discussions with the FE team, we are just going to reduce the page size for the transfers API for now and leave the default page size for other APIs.

Once the Palazzo release has gone out, we can start looking at reducing the default page size to something smaller as we will have more time to check for breaks.